examples/auto_retry/test_app.py

```diff
@@ -36,4 +38,13 @@ def test_foo(self):
 
         server.execute(app_client)
 
-        self.assertEqual(len(servicer.responses), 1)
+        # Seed(2) used in the app outputs 0, 0, 0, 2, 1, 5. So we expect 6
+        # calls, including 5 retries.
+        for i in range(6):
+            server.execute(app_client)
+        self.assertEqual(len(servicer.responses), 6)
+
+        statuses = [r["response"].status for r in servicer.responses]
+        self.assertEqual(
+            statuses, [status_pb.STATUS_TEMPORARY_ERROR] * 5 + [status_pb.STATUS_OK]
+        )
```

Comment:
no `assert`?

Reply:
I don't have a preference for one style over the other. Since the rest of the test had already been written that way, I kept it consistent.